### ISSUE TITLE
0010628:
Calender print shows only time range 0-11 o'clock

### ISSUE THREAD

**Issue by Gloirin** [tine20/tine20#5297]
**Reported by it-mvz on 5 Jan 2015 10:16**

**Version:** Koriander (2014.09.5)

When you print the calender, you get only the time range printed from 0-11 o&#039;clock and not the full calendar or at least the interesting time (working hours). See attached scan of a calendar print

**Steps to reproduce:** - Print calender (week view)

**Comment by Gloirin** [comment]
Related to #5612

**Comment by Gloirin** [comment]
**Comment posted by cweiss on 19 Aug 2015 10:06**

just fixed it.

**Comment by Gloirin** [comment]
**Comment posted by it-mvz on 24 Sep 2015 08:27**

We updated to Elena 2015.07.4, but the problem still exists - even if the time range now shows from 0-16 o&#039;clock instead of 0-11 before. There&#039;s a second page now, but this one is usless because it is compressed and unreadable. I filled a new bug report for that (#11340). I don&#039;t know if this is a new issue or still this one, that wasn&#039;t fully solved.
